cinema-soft/src/test/java/softserve/academy/cinemasoft/integration/MovieControllerIT.java

```diff
@@ -42,33 +48,51 @@
 
     @Before
     public void setUp() throws Exception {
-
-        this.directorsUrl = new URL("http://localhost:" + port + "/" + "searchByDirectorName/" + "Dir");
         fillDatabase();
     }
 
     @Test
-    public void getDirectors() {
+    public void getDirectors() throws MalformedURLException {
+        this.directorsUrl = new URL("http://localhost:" + port + "/" + "searchByDirectorName/" + "Dir");
         ResponseEntity<String> response = template.getForEntity(directorsUrl.toString(), String.class);
         assertThat(response.getBody().contains("[{\"title\":\"Test Title\",\"director\":\"Test Director\"}]"), equalTo(true));
     }
 
-    private void fillDatabase() {
+    @Test
+    public void getNoDirectors() throws MalformedURLException {
+        this.directorsUrl = new URL("http://localhost:" + port + "/" + "searchByDirectorName/" + "Dir");
+        ResponseEntity<String> response = template.getForEntity(directorsUrl.toString(), String.class);
+        assertThat(response.getBody().contains("[]"), equalTo(false));
+    }
 
+    @Test
+    public void getMoviesRating() throws MalformedURLException {
+        this.ratingUrl = new URL("http://localhost:" + port + "/" + "ratingGreaterOrEqualThan/" + "3");
+        ResponseEntity<String> response = template.getForEntity(ratingUrl.toString(), String.class);
+        assertThat(response.getBody().contains("[{\"title\":\"Test Title\",\"rating\":3.1}]"), equalTo(true));
+    }
+
+    @Test
+    public void getLessMoviesRating() throws MalformedURLException {
+        this.ratingUrl = new URL("http://localhost:" + port + "/" + "ratingGreaterOrEqualThan/" + "3");
```

Comment:
Use a value that you know you don't have to check that the method will return an empty list.

Reply:
ok.